src/edgepi/digital_output/edgepi_digital_output.py

```diff
@@ -22,11 +25,22 @@ class EdgePiDigitalOutput():
         DoutPins.DOUT7 : GpioPins.AO_EN7,
         DoutPins.DOUT8 : GpioPins.AO_EN8,
     }
+    _dout_dac_pair = {
+        DoutPins.DOUT1 : Ch.AOUT1,
+        DoutPins.DOUT2 : Ch.AOUT2,
+        DoutPins.DOUT3 : Ch.AOUT3,
+        DoutPins.DOUT4 : Ch.AOUT4,
+        DoutPins.DOUT5 : Ch.AOUT5,
+        DoutPins.DOUT6 : Ch.AOUT6,
+        DoutPins.DOUT7 : Ch.AOUT7,
+        DoutPins.DOUT8 : Ch.AOUT8,
+    }
     def __init__(self):
         # To limit access to input functionality, using composition rather than inheritance
         self.gpio = EdgePiGPIO()
+        self.dac = EdgePiDAC()
 
-    def digital_output_state(self, pin_name: DoutPins = None, state: bool = None):
+    def digital_output_state(self, pin_name: DoutPins = None, state: DoutTriState = None):
```

Comment:
can we rename this function to something like `set_dout_state`?

Reply:
do you want it dout instead of digital output? we wanted to be explicit on the name.

Reply:
c64b58fe15dd94139db7e678f51a964dd20c790c

Reply:
up to you